test/unit/test_loader.py

```diff
@@ -14,12 +15,12 @@ class TestCKANDatastoreBase(unittest.TestCase):
     def setUp(self):
         self.pipeline = pl.Pipeline(
             'test', 'Test',
-            settings_file=os.path.join(HERE, '../mock/first_test_settings.json'),
             log_status=False
         )
-        self.ckan_config = self.pipeline.parse_config_piece(
-            'loader', 'ckan'
-        )
+        settings_file = os.path.join(HERE, '../mock/first_test_settings.json')
+        with open(settings_file) as f:
```

Comment:
Is this how you would implement this in production? If so, I'm really not sure what the benefit of dumping the config file is.